website/docs/r/apimanagement_workspace.html.markdown

```diff
@@ -0,0 +1,72 @@
+---
+subcategory: "API Management"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_api_management_workspace"
+description: |-
+  Manages an API Management Workspace.
+---
+
+# azurerm_api_management_workspace
+
+Manages an API Management Workspace.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "East US"
```

Comment:
To keep consistent with existing resources, the location should be `"West Europe"`.